src/System.ServiceModel.Duplex/tests/System.ServiceModel.Duplex.Tests.csproj

```diff
@@ -23,9 +23,6 @@
     <None Include="project.json" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="$(WcfSourcePkgProj)">
-      <Name>System.Private.ServiceModel</Name>
-    </ProjectReference>
```

Comment:
Updated.
@ericstj could you confirm, not just the façade pkgproj but also the S.P.SM.pkgproj?